js/details.js

```diff
@@ -0,0 +1,50 @@
+const flags = document.getElementById('flags')
+const query = new URLSearchParams(window.location.search)
+const params = query.get('name')
+console.log(params)
```

Comment:
Remove console.logs after you are done with them, otherwise you polute the browser console